src/core/types/assets.ts

```diff
@@ -41,6 +41,10 @@ export interface ParsedAsset {
     networks: { [id in ChainId]?: { bridgeable: boolean } };
   };
   transferable?: boolean;
+  platformValue?: {
```

Comment:
Any reason we are adding this, versus using the existing `balance: { amount, display }` structure in `ParsedUserAsset`?